tests/test_checkout.py

```diff
@@ -68,6 +68,118 @@ def test_merge_diverged_conflict(self):
         pass
 
 
-@pytest.mark.skip(reason='not implemented')
 class TestTimeTravel(object):
-    pass
+
+    def test_merge_data_removal(self, written_repo, array5by7, get_history):
+        co = written_repo.checkout(write=True)
+        co.datasets['_dset']['1'] = array5by7
+        co.datasets['_dset']['2'] = array5by7
+        co.metadata.add('a1', 'b1')
+        co.metadata.add('a2', 'b2')
+        co.commit('Commit with two data point')
+        co.close()
+        branch = written_repo.create_branch('testbranch')
+        co = written_repo.checkout(write=True, branch_name=branch)
+        co.datasets['_dset'].remove('2')
+        co.metadata.remove('a2')
+        co.commit('removing one data point')
+        co.close()
+        written_repo.merge('removal test merge', 'master', branch)
+        co = written_repo.checkout()
+        assert len(co.datasets['_dset']) == 1
+        assert '1' in co.datasets['_dset'].keys()
+        assert list(co.metadata.keys()) == ['a1']
+
+        # TODO: better way to get history? right now, only possible
+        # with write checkout?
+        co = written_repo.checkout(write=True)
+        history = get_history(co._refenv, co._branchenv)
```

Comment:
Please don't use the `get_history` fixture: https://github.com/tensorwerk/hangar-py/pull/25#discussion_r281063460. 

this is not just possible from a write checkout. the `repo.log()` function (optionally with `return_contents=True` for programmatic access).